app/controllers/triannon/auth_controller.rb

```diff
@@ -5,64 +5,56 @@ module Triannon
   class AuthController < ApplicationController
     include RdfResponseFormats
 
-    AUTH_EXPIRY  =   60 # seconds
-    TOKEN_EXPIRY = 3600 # seconds
+    # HTTP request methods accepted by /auth/login
+    LOGIN_ACCEPT = 'OPTIONS, GET, POST'
 
     # OPTIONS /auth/login
     def options
```

Comment:
@azaroth42 - does the current implementation return the 'CORS headers' required?  Should we investigate whether rack-cors should be used?